tools.dm

```diff
@@ -0,0 +1,58 @@
+// Tool types, if you add new ones please add them to /obj/item/debug/omnitool in code/game/objects/items/debug_items.dm
+#define TOOL_CROWBAR "crowbar"
```

Comment:
This file is a duplicate of `code/_DEFINES/tools.dm`. Please delete it

Reply:
yeah my bad lol, somehow i deleted it and pasted everything again